src/models/ssh-key.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) The Arvados Authors. All rights reserved.
+//
+// SPDX-License-Identifier: AGPL-3.0
+
+import { Resource } from '~/models/resource';
+
+export interface SshKey {
+    name: string;
+    keyType: KeyType;
+    authorizedUserUuid: string;
+    publicKey: string;
+    expiresAt: string;
+}
+
+export interface SshKeyCreateFormDialogData {
```

Comment:
Until now we kept dialog data interfaces inside or next to corresponding actions file. Why the change?